frontend/src/services/users-api/users-api.service.ts

```diff
@@ -39,6 +41,20 @@ class UsersApi {
     );
   }
 
+  public getMentor({
+    id,
+    courseId,
+  }: UserGetMentorRequestParamsDto): Promise<MenteesToMentorsResponseDto> {
+    return this.#http.load(
+      `${this.#apiPrefix}${ApiPath.USERS}${
+        UsersApiPath.ROOT
+      }${id}/courses/${courseId}/mentor`,
```

Comment:
```suggestion
      }${id}/courses/${courseId}/mentor`,
```
should be enum values instead of strigs